src/Attributes/CastWith.php

```diff
@@ -15,7 +15,7 @@ public function __construct(
         public string $casterClass,
         mixed ...$args
     ) {
-        if (! class_implements($this->casterClass, Caster::class)) {
+        if (! is_subclass_of($this->casterClass, Caster::class)) {
```

Comment:
I think we need the third parameter to be `true` here, because the first one is a string.
```suggestion
        if (! is_subclass_of($this->casterClass, Caster::class, true)) {
```

Reply:
What for? There is true by default.

Reply:
Oh, you're right!